ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -75,8 +76,16 @@ def install_ocs_operator(self):
             self.do_click(self.dep_loc["enable_console_plugin"], enable_screenshot=True)
         self.do_click(self.dep_loc["click_install_ocs_page"], enable_screenshot=True)
         if self.operator is ODF_OPERATOR:
-            time.sleep(60)
-            self.refresh_page()
+            time.sleep(90)
```

Comment:
waiting for a fixed time period may result in either not having enough time to wait or redundant time we continue waiting even after the state is achieved.
Could you please change this to wait for the resource mechanism?